pkg/reconciler/build/build.go

```diff
@@ -215,6 +215,7 @@ func (c *Reconciler) reconcile(ctx context.Context, build *buildapi.Build) error
 		build.Status.LatestAttestationImage = attestDigest
 		build.Status.Stack.RunImage = buildMetadata.StackRunImage
 		build.Status.Stack.ID = buildMetadata.StackID
+		build.Status.Lifecycle.Version = buildMetadata.LifecycleVersion
```

Comment:
Somewhere along the line I went from checking Version to checking Id... I will look into it